dashboards/aerospike/aerospike-prometheus-overview.json

```diff
@@ -17,207 +18,500 @@
       "templateVariable": "Namespace"
     }
   ],
-  "displayName": "Aerospike Prometheus Overview",
-  "labels": {},
   "mosaicLayout": {
-    "columns": 12,
+    "columns": 48,
     "tiles": [
       {
+        "width": 8,
+        "height": 8,
+        "widget": {
+          "title": "Uptime (seconds)",
+          "scorecard": {
+            "blankView": {},
+            "thresholds": [],
+            "timeSeriesQuery": {
+              "outputFullDuration": true,
+              "prometheusQuery": "aerospike_node_stats_uptime{${Cluster},${Location},${Namespace}}",
+              "unitOverride": ""
+            }
+          }
+        }
+      },
+      {
+        "xPos": 8,
+        "width": 8,
+        "height": 8,
+        "widget": {
+          "title": "Active Threads",
+          "scorecard": {
+            "blankView": {},
+            "thresholds": [],
+            "timeSeriesQuery": {
+              "outputFullDuration": true,
+              "prometheusQuery": "aerospike_node_stats_threads_pool_active{${Cluster},${Location},${Namespace}}",
```

Comment:
The top row of scorecards all need aggregations so that if you have multiple aerospike instances in multiple clusters in the same project, the multiple time series get condensed down to one.

I think we want `max()` for uptime and `sum()` for the other scorecards.